api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3211,6 +3211,13 @@ message Identity {
     (gogoproto.jsontag) = "allowed_resource_ids,omitempty",
     (gogoproto.nullable) = false
   ];
+  // MFAVerifiedSessionExpires is the the hard deadline of a session when this Identity was
+  // confirmed immediately after an MFA check.
+  google.protobuf.Timestamp MFAVerifiedSessionExpires = 23 [
```

Comment:
I think we can do a better job with the name. Remember, the root issue is that the "disconnect expired cert" option doesn't work with session MFA. This is not really a session expiry, because there are valid use cases where a cert expires but the session is allowed to continue.

How about

Reply:
(Looks like you forgot to include your suggestion)

Yeah good point. How about `IssuingCertExpires`?

Reply:
73889219589f717c33193070365f12b631ae64c3